doc/docs/1.9.x/concepts/advanced-concepts/deferred_processing.md

```diff
@@ -144,7 +144,7 @@ To configure a staging branch, complete the following steps:
    because commits in Pachyderm are generally additive, so processing
    the HEAD commit also processes data from previous commits.
 
-![deffered processing](../assets/images/deferred_processing.gif)
+![deffered processing](../../assets/images/deferred_processing.gif)
```

Comment:
Might want to correct the spelling on the alt tag.

![deferred processing](../../assets/images/deferred_processing.gif)

Reply:
This appears to be in the wrong path to fix the broken link at https://docs.pachyderm.com/latest/how-tos/deferred_processing, or am I misunderstanding how the docs get built?

Reply:
https://deploy-preview-4544--pachyderm-docs.netlify.com/latest/how-tos/deferred_processing/ this should do it. The changes will go live after this commit is merged.